### ISSUE TITLE
[Bug]: Document accessible usage of Tabs when not leveraging "sp-tabs-panel"

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#2729]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

Tabs

### Expected behavior

Accessibility usage of Tabs without Panel should be documented

### Actual behavior

There is not include.

**Comment by Westbrook** [comment]
Especially useful when rendering into a "single panel" as opposed to multiple panels with dynamism.

**Comment by castastrophe** [comment]
We have this work tracked now in Jira so I'm going to close. Please do reach out if you still need a status update on this feature.
